content/en/user-guide/aws/rds/index.md

```diff
@@ -1,83 +1,44 @@
 
-{{< alert title="Note" >}}
-Please be aware that MySQL images for `arm64` are only available for newer versions. Please check the [MySQL Docker Hub repository](https://hub.docker.com/_/mysql) for details on the availability.
-{{< /alert >}}
-
-Please note that the `MasterUserPassword` defined for the database cluster/instance will be used as the `MYSQL_ROOT_PASSWORD` environment for user `root` in the MySQL container. The user for `MasterUserName` will use the same password, and will have full access to the defined database.
+LocalStack supports RDS via the Pro/Team offering, allowing you to use the RDS APIs in your local environment to create and manage RDS clusters and instances for testing & integration purposes. The supported APIs are available on our [API coverage page](https://docs.localstack.cloud/references/coverage/coverage_rds/), which provides information on the extent of RDS's integration with LocalStack.
 
-DB Snapshots are currently not supported for MySQL.
+## Getting started
 
-### MSSQL Engine
+This guide is designed for users new to RDS and assumes basic knowledge of the AWS CLI and our [`awslocal`](https://github.com/localstack/awscli-local) wrapper script.
 
-{{< alert title="Note" >}}
-In order to use MSSQL databases, you need to explicitly accept the [Microsoft SQL Server End-User Licensing Agreement (EULA)](https://hub.docker.com/_/microsoft-mssql-server) by setting `MSSQL_ACCEPT_EULA=Y` in the LocalStack container environment.
-
-Please note that MSSQL does not yet have official support for `arm64`. 
-{{< /alert >}}
+Start your LocalStack container using your preferred method. We will demonstrate the following with the AWS CLI:
 
-For the MSSQL engine, the database server is started in a new docker container using the `latest` image.
+1. Creating an RDS cluster.
+2. Generating a `SecretsManager` secret containing the database password.
+3. Executing a basic `SELECT 123 query` through the RDS Data API.
 
-DB Snapshots are currently not supported for MSSQL.
+LocalStack's RDS implementation also supports the [RDS Data API](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/data-api.html), which allows executing data queries against RDS clusters over a JSON/REST interface. 
 
-## End-to-end example (Postgres)
+### Create an RDS cluster
 
-The local RDS service also supports the [RDS Data API](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/data-api.html), which allows executing data queries against RDS clusters over a JSON/REST interface. 
+To create an RDS cluster, you can use the [`CreateDBCluster`](https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_CreateDBCluster.html) API. The following command creates a new cluster with the name `db1` and the engine `aurora-postgresql`. The cluster will be created with a single instance, which will be used as the master instance. The instance will be created with the default username `test` and password `test`.
```

Comment:
> The instance will be created with the default username `test` and password `test`
IMO we should not outline this here, for a few reasons. 

- The sample below actually sets a username + password (which is different from the default) and it might confuse people. 
- In general it's not possible to leave the fields blank for the real AWS, so I think we shouldn't promote non-parity behavior.
- For some databases (MySQL + MSSQL) we spin up the official images which will not work with the defaults (for MySQL this just changed recently, and we rely on the user setting a valid user-password combination).